tests/cpu/transforms/builtins/ProPhotoRGB_tests.cpp

```diff
@@ -0,0 +1,293 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
```

Comment:
Usually I really appreciate it when people add tests, but unfortunately a lot of this is either not necessary or nonsensical.  I'm guessing AI was used extensively for these?  

I would just delete this module and add a few lines in BuiltinTransform_tests.cpp, like all the other builtins.